projects/librdkafka.cmake

```diff
@@ -2,18 +2,18 @@
 # librdkafka
 ########################################
 xpProOption(librdkafka)
-set(KAFKA_VER 16.03.1)
-set(KAFKA_REPO https://github.com/distributePro/librdkafka)
-set(KAFKA_REPO_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/librdkafka_repo)
-set(PRO_KAFKA
+set(LIBRDKAFKA_VER 16.03.1)
+set(LIBRDKAFKA_REPO https://github.com/distributePro/librdkafka)
```

Comment:
For whatever reason it wasn't generating stuff in the README for librdkafka unless I changed it - I think it has something to do with the project name needing to match the PRO_LIBRDKAFKA, so I probably didn't need to add LIBRD to all of these, but by the time I figured out what really had fixed it I had already changed them all.